terraform/modules/monitoring/grafana_task_definition.tf

```diff
@@ -4,7 +4,7 @@ locals {
     essential   = true,
     environment = [for key, value in local.grafana_environment_variables : { name : key, value : tostring(value) }],
     healthCheck = {
-      command     = ["/bin/bash", "-c", "curl -f http://localhost:${var.grafana_port} || exit 1"]
+      command     = ["/bin/bash", "-c", "wget -q -O - http://localhost:3000/api/health || exit 1"]
```

Comment:
Ah, does the base image not come with curl then? I guess we can live with having two different utilities for healthchecks. You might still want to substitute the port variable though.

Reply:
no curl, unfortunately :-(